packages/styles/src/link/index.css

```
@@ -0,0 +1,82 @@
+/**
+ * Link styles
+ * @variant {small | medium | large} syn-link The size of the link
+ * @boolean { false } syn-link--quiet Applies the quiet link styling
+ * @boolean { false } syn-link--disabled Applies the disabled link styling
+ */
+.syn-link {
+  align-items: center;
+  color: var(--syn-link-color);
+  display: inline-flex;
+  font: var(--syn-body-medium-regular);
+  gap: var(--syn-spacing-x-small);
+}
+
+/* Use inline text styling when there is no syn-icon */
+.syn-link:not(:has(syn-icon)) {
+  display: inline;
+}
+
+.syn-link:not(.syn-link--disabled):hover {
+  color: var(--syn-link-color-hover);
+}
+
+.syn-link:not(.syn-link--disabled):active {
+  color: var(--syn-link-color-active);
+}
+
+.syn-link:not(.syn-link--disabled):focus-visible {
+  outline: var(--syn-focus-ring);
+  outline-offset: var(--syn-focus-ring-offset);
+}
+
+/* Overrides for smaller links */
+.syn-link--small {
+  font: var(--syn-body-small-regular);
+  gap: var(--syn-spacing-2x-small);
+}
+
+/*
+ * Overrides for medium links.
+ * Note: The default styles are already set for medium links, just here for completeness
+ */
+.syn-link--medium {
+  font: var(--syn-body-medium-regular);
+  gap: var(--syn-spacing-x-small);
+}
```

```suggestion
.syn-link--medium {
  font: var(--syn-body-medium-regular);
  gap: var(--syn-spacing-x-small);
}
```

We should not use `--syn-body-medium-regular` but `--syn-font-size-medium` here (as done in design).

Otherwise this would lead to:
```html
<div class="font-bold">
Reprehenderit aliquip pariatur <a href="syn-link"> proident nulla fugiat Lorem ex.
</div>
```

Here the link would not be bold anymore.

---

You are right, I did not think about this. Adjusted.